x-pack/plugins/watcher/public/application/sections/watch_status_page/components/execution_history_panel.tsx

```diff
@@ -202,16 +266,38 @@ export const ExecutionHistoryPanel = () => {
             defaultMessage: 'Name',
           }),
           sortable: true,
-          truncateText: true,
+          truncateText: false,
         },
         {
           field: 'state',
-          name: i18n.translate('xpack.watcher.sections.watchHistory.watchActionStatusTable.state', {
-            defaultMessage: 'State',
-          }),
+          name: (
+            <EuiToolTip
+              content={i18n.translate(
+                'xpack.watcher.sections.watchHistory.watchActionStatusTable.state.tooltipText',
+                {
+                  defaultMessage: 'OK, acknowledged, throttled, or error.',
```

Comment:
How does this related to acknowledge, acknowledgeable, throttled, and failed.  Should these two messages be using the same words?